readthedocs.yml

```diff
@@ -7,3 +7,10 @@ build:
   os: "ubuntu-20.04"
   tools:
     python: "mambaforge-4.10"
+  jobs:
+    post_checkout:
+      # This is necessary for setuptools_scm to properly read the tags. The default
+      # depth of 50 often leads to 'assert version is not None' AssertionError during
+      # the pip install build process. Alternatively, we could use
+      # 'git fetch --unshallow', but that is rather intensive.
+      - git fetch --depth 1000
```

Comment:
My understanding is that we compute the version by looking through the git history to find the latest matching git tag. The default commands (see the RTD CI logs) download a truncated history. This command runs afterwards to download more. I chose 1000 because it's way more than I think we would ever need, and way smaller than the whole commit history. What Ida at don't understand is why it was no prob w versioneer

Reply:
Are you still concerned about this? I think `--depth 1000` is better than `--unshallow` (i.e. depth=\infty) due to bandwidth/CI time considerations, and 1000 is way more than we could conceivably need.

Reply:
No, that's not a real concern.

Reply:
I have a vague suspicion about what might be going on here...

In case I'm working from a fork, then it's possible that my fork might not have all the tags. (Git is sometimes a bit weird about synchronizing tags, although mainly in the push direction.) Then `setuptools_scm` may not have the info to compute the version.

Typing this out, it doesn't make much sense, but at some point during testing I somehow ended up installing a very "old" version of aesara simply because the tags were old. I don't have any actionable ideas, this is just something to think about.